### PR TITLE
request merge

### PR DESCRIPTION
**Pull request by GitHubYucky**
add sub message
please confirm

### CODE REVIEW

File: Earth/Form1.cs [hunk 1]
```diff
@@ -5,6 +5,8 @@ public partial class Form1 : Form
         public Form1()
         {
             InitializeComponent();
+            MessageBox.Show("yucky yucky");
+            MessageBox.Show("yucky2 yucky2");
```
Comment (hogemoge333) [inline Earth/Form1.cs]: sub3 message is required

**Review by hogemoge333 — CHANGES_REQUESTED** [review]
confirm comment

File: Earth/Form1.cs (hunk 1 above)
Comment (GitHubYucky) [reply Earth/Form1.cs]: fixed

**Comment by GitHubYucky** [comment]
fix review please

**Review by hogemoge333 — APPROVED** [review]
ok

**Comment by hogemoge333** [comment]
ok

**Pull request merged**